HorizonEditor/src/EditorLayer.cpp

```diff
@@ -90,21 +87,22 @@ void EditorLayer::onUpdate(Hzn::TimeStep ts)
 
 	auto viewportSize = m_ViewportBounds[1] - m_ViewportBounds[0];
 
-	if(0 < mousePos.x && mousePos.x < viewportSize.x  && 0 < mousePos.y && mousePos.y < viewportSize.y)
-		HZN_INFO("{0}, {1}", mousePos.x, mousePos.y);
-
+	if (0 < mousePos.x && mousePos.x < viewportSize.x && 0 < mousePos.y && mousePos.y < viewportSize.y)
+	{
+		/*HZN_INFO("{0}, {1}", mousePos.x, mousePos.y);*/
```

Comment:
Why is the code within the comment unnecessary

Reply:
This is for the next update.